patternbuilder.admin.inc

```diff
@@ -1,4 +1,5 @@
 <?php
+
```

Comment:
This gets flagged by phpcs for the Drupal standard "Drupal.WhiteSpace.OpenTagNewline.BlankLine"